samples/Mvc.ControllerForMultipleAreasSample/MultipleAreasControllerConvention.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+
+namespace ControllerForMultipleAreasSample
+{
+    public class MultipleAreasControllerConvention : IApplicationModelConvention
+    {
+        public MultipleAreasControllerConvention()
+        {
+            ControllerModels = new List<ControllerModel>();
+        }
+
+        public IList<ControllerModel> ControllerModels { get; set; }
```

Comment:
This should be a local variable, there's no need for it to exist outside the scope of `Apply`